regolith/schemas.py

```diff
@@ -3798,17 +3872,22 @@
                                                 "in these groups their names or id's can be added explicitly to the list",
                                  "required": False,
                                  "type": "list"},
-                    "status": {"description": f"status of the milestone. "
-                                              f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
+                    "status": {"description": "status of the milestone. "
+                                              "Allowed values are 'proposed', 'started', 'converged', 'finished', 'cancelled', 'paused'",
                                "required": False,
                                "type": "string",
                                "eallowed": PROJECTUM_STATUS},
                     "type": {"description": f"type of milestone deliverable. "
                                             f"Allowed values are {', '.join(MILESTONE_TYPE)}",
                              "required": False,
                              "type": "string",
-                             "eallowed": MILESTONE_TYPE}
-
+                             "eallowed": MILESTONE_TYPE},
+                    "end_date": {"description": "end date of milestone, yyyy-mm-dd",
+                                 "required": False,
+                                 "anyof_type": ["date", "string"]},
+                    "identifier": {"description": "label of milestone",
```

Comment:
See comments, but I think this needs to be removed unless we can figure out what it is used for.

Reply:
Done: deleted